app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -135,13 +135,15 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     private LeanbackOverlayFragment leanbackOverlayFragment;
 
     // Subtitle fields
-    private static final int SUBTITLE_PADDING = 8;
+    private static final int SUBTITLE_PADDING = 20;
     private static final long SUBTITLE_RENDER_INTERVAL_MS = 50;
     private SubtitleTrackInfo subtitleTrackInfo;
     private int currentSubtitleIndex = 0;
-    private int subtitlesSize = KoinJavaComponent.<UserPreferences>get(UserPreferences.class).get(UserPreferences.Companion.getDefaultSubtitlesSize());
+    private final int subtitlesSize = KoinJavaComponent.<UserPreferences>get(UserPreferences.class).get(UserPreferences.Companion.getDefaultSubtitlesSize());
```

Comment:
Put `KoinJavaComponent.<UserPreferences>get(UserPreferences.class)` before use for better readability